packages/applicant/src/pages/submissions/[submissionId]/summary.page.test.tsx

```diff
@@ -23,6 +41,244 @@ jest.mock(
+  fieldTitle: 'Enter the amount',
+  hintText:
+    'This is the official name of your organisation. It could be the name that is registered with Companies House or the Charities Commission',
+  responseType: 'MultipleSelection',
+  validation: {
+    mandatory: true,
+    greaterThanZero: true,
+  },
+  multiResponse: ['test', 'test2'],
+};
+const eligibility: QuestionType = {
+  questionId: 'ELIGIBILITY',
+  profileField: 'ORG_AMOUNT',
+  fieldTitle: 'Enter the amount',
+  hintText:
+    'This is the official name of your organisation. It could be the name that is registered with Companies House or the Charities Commission',
+  responseType: 'YesNo',
+  validation: {
+    mandatory: true,
+    greaterThanZero: true,
+  },
+  response: 'Yes',
+};
+const propsWithAllValues: ApplicationDetailsInterface = {
+  grantSchemeId: 'schemeId',
+  grantApplicationId: 'string',
+  grantSubmissionId: 'string',
+  applicationName: 'Name of the grant being applied for',
+  submissionStatus: 'IN_PROGRESS',
+  sections: [
+    {
+      sectionId: 'ELIGIBILITY',
+      sectionTitle: 'Eligibility',
+      sectionStatus: 'COMPLETED',
+      questionIds: [eligibility].map((q) => q.questionId),
+      questions: [eligibility],
+    },
+    {
+      sectionId: 'ORGANISATION_DETAILS',
+      sectionTitle: 'Your Organisation',
+      sectionStatus: 'IN_PROGRESS',
+      questionIds: [shortAnswer].map((q) => q.questionId),
+      questions: [shortAnswer],
+    },
+    {
+      sectionId: 'FUNDING_DETAILS',
+      sectionTitle: 'Funding',
+      sectionStatus: 'IN_PROGRESS',
+      questionIds: [numeric].map((q) => q.questionId),
+      questions: [numeric],
+    },
+    {
+      sectionId: 'ESSENTIAL',
+      sectionTitle: 'Essential Information',
+      sectionStatus: 'COMPLETED',
+      questionIds: [shortAnswer].map((q) => q.questionId),
+      questions: [shortAnswer],
+    },
+    {
+      sectionId: 'NON-ESSENTIAL',
+      sectionTitle: 'Non Essential Information',
+      sectionStatus: 'IN_PROGRESS',
+      questionIds: [shortAnswer, numeric].map((q) => q.questionId),
+      questions: [shortAnswer, numeric],
+    },
+  ],
+};
+
+mockServiceMethod(
+  spiedGetMandatoryQuestionBySubmissionId,
+  mockMandatoryQuestionDto
+);
+(getQuestionById as jest.Mock).mockImplementation(
+  async (_submissionId, _sectionId, questionId, _jwt) => {
+    return {
+      question: [numeric, eligibility, shortAnswer].find(
+        (q) => q.questionId === questionId
+      ),
+    };
+  }
+);
+
+describe('getServerSideProps', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+  it('should return a redirect to grant-is-closed when submission is REMOVED ', async () => {
+    (getApplicationStatusBySchemeId as jest.Mock).mockResolvedValue('REMOVED');
+    (getSubmissionById as jest.Mock).mockReturnValue(propsWithAllValues);
+    (getJwtFromCookies as jest.Mock).mockReturnValue('testJwt');
+    (hasSubmissionBeenSubmitted as jest.Mock).mockReturnValue(false);
```

Comment:
You can use the `jest.mocked` utility to get a properly typed mock function and avoid having cast these multiple times

```
const mockGetApplicationStatusByShchemeId = jest.mocked(getApplicationStatusBySchemeId;
etc
```